### PR TITLE
Add debug option for less verbose output

### PR DESCRIPTION
**Pull request by jonjaques**

### CODE REVIEW

**Comment by jgoz** [comment]
:+1: for the concept, but I think `verbose` might be a better name for the flag. To me, `debug` has implications for compiler output rather than the log output.

**Comment by andreypopp** [comment]
+1 for `verbose` as `debug` maybe confusing with `webpack -d` which provides debugging tools for code, not for the tool itself.

**Comment by andreypopp** [comment]
@jonjaques btw. are you ok if I add you as a collaborator to this repo?

File: TypeScriptWebpackHost.js [hunk 1]
```diff
@@ -106,7 +107,7 @@ TypeScriptWebpackHost.prototype.getDefaultLibFilename = function getDefaultLibFi
  * Implementation of TypeScript Language Services Host interface.
  */
 TypeScriptWebpackHost.prototype.log = function log(message) {
-  return console.log(message);
+  return this.options.debug ? console.log(message) : void 0;
```
Comment (andreypopp) [inline TypeScriptWebpackHost.js]: Let's make it

```
if (this.options.debug) {
  console.log(message);
}
```

**Comment by jonjaques** [comment]
@andreypopp I'm still digging into the webpack ecosystem, so I will probably have lots of questions - but I would love to contribute. Will get this commit patched up in a bit.

**Pull request merged**